### DISCUSSION TITLE
Unable to delete joint unique index

### DISCUSSION THREAD

**Discussion by baijingetangzichen** [sqlalchemy/sqlalchemy#10172] [Usage Questions]
### Describe the bug

class DataSetPretreatmentServiceModel(db.Model):

    id = db.Column(db.Integer, autoincrement=True, primary_key=True, comment="ID")
    name = db.Column(db.String(64), comment="服务名称")
    owner = db.Column(db.Text, nullable=True, comment="服务制作者")
    version = db.Column(db.String(32), nullable=True, comment="服务版本")
    description = db.Column(db.String(128), nullable=True, comment="描述")
    created_on = db.Column(db.DateTime, default=lambda: datetime.datetime.now(), comment="创建时间")

    __tablename__ = "comm_data_ann_preprocess_service"
   # __table_args__ = (
   #     db.UniqueConstraint("name", "owner", "created_on", name="ix_preprocessing_service_unique_create_on"),
   # )

sqlalchemy.exc.NotSupportedError: (psycopg2.errors.FeatureNotSupported) unimplemented: cannot drop UNIQUE constraint "ix_preprocessing_service_unique_create_on" using ALTER TABLE DROP CONSTRAINT, use DROP INDEX CASCADE instead
HINT:  You have attempted to use a feature that is not yet implemented.

Please check the public issue tracker to check whether this problem is
already tracked. If you cannot find it there, please report the error
with details by creating a new issue.

If you would rather not post publicly, please contact us directly
using the support form.

We appreciate your feedback.

### SQLAlchemy Version in Use

1.4.39

### DBAPI (i.e. the database driver)

psycopg2=2.8.6  

### Database Vendor and Major Version

postgresql 13.0.0

### Python Version

python 3.9.12

### Operating system

linux

### To Reproduce

```python
class DataSetPretreatmentServiceModel(db.Model):

    id = db.Column(db.Integer, autoincrement=True, primary_key=True, comment="ID")
    name = db.Column(db.String(64), comment="服务名称")
    owner = db.Column(db.Text, nullable=True, comment="服务制作者")
    version = db.Column(db.String(32), nullable=True, comment="服务版本")
    description = db.Column(db.String(128), nullable=True, comment="描述")
    created_on = db.Column(db.DateTime, default=lambda: datetime.datetime.now(), comment="创建时间")

    __tablename__ = "comm_data_ann_preprocess_service"
   # __table_args__ = (
   #     db.UniqueConstraint("name", "owner", "created_on", name="ix_preprocessing_service_unique_create_on"),
   # )



sqlalchemy.exc.NotSupportedError: (psycopg2.errors.FeatureNotSupported) unimplemented: cannot drop UNIQUE constraint "ix_preprocessing_service_unique_create_on" using ALTER TABLE DROP CONSTRAINT, use DROP INDEX CASCADE instead
HINT:  You have attempted to use a feature that is not yet implemented.

Please check the public issue tracker to check whether this problem is
already tracked. If you cannot find it there, please report the error
with details by creating a new issue.

If you would rather not post publicly, please contact us directly
using the support form.

We appreciate your feedback.
```

### Error

```
# qlalchemy.exc.NotSupportedError: (psycopg2.errors.FeatureNotSupported) unimplemented: cannot drop UNIQUE constraint "ix_preprocessing_service_unique_create_on" using ALTER TABLE DROP CONSTRAINT, use DROP INDEX CASCADE instead
HINT:  You have attempted to use a feature that is not yet implemented.
```

**Comment by CaselIT** [comment]
Hi

It's not clear what you are trying to do.
This error comes from the database, so you will have to look at its documentation to see how to remove it. The error tesmxt is giving suggestions
